src/main/kotlin/net/leanix/githubagent/services/GitHubAuthenticationService.kt

```diff
@@ -34,6 +37,15 @@ class GitHubAuthenticationService(
         private val logger = LoggerFactory.getLogger(GitHubAuthenticationService::class.java)
     }
 
+    fun refreshTokens() {
+        generateJwtToken()
+        val jwtToken = cachingService.get("jwtToken")
+        generateAndCacheInstallationTokens(
+            gitHubClient.getInstallations("Bearer $jwtToken"),
+            jwtToken.toString()
+        )
+    }
```

Comment:
`generateJwtToken()` should return the token as well.

Also, in `generateJwtToken()`, I think we should `verifyJwt` first and `cachingService.set("jwtToken", jwt.getOrThrow(), JWT_EXPIRATION_DURATION)` after. (Not part of this PR changes though)